package.json

```diff
@@ -17,7 +23,7 @@
   "license": "Apache-2.0",
   "scripts": {
     "automated-release": "release-it -VV --config ./release/.release-it.json",
-    "build": "run-p --aggregate-output build-core build-bundle build-standalone build-stylesheets build:es:bundle build:es:bundle:core",
+    "build": "run-p --aggregate-output build-stylesheets build-core build-bundle build-standalone build:es:bundle build:es:bundle:core",
```

Comment:
ok

Reply:
We can still use `rimraf` to clear it manually so that it's explicit that this is happening
```json
"build": "npm run build-stylesheets && rimraf ./dist/swagger-ui.js ./dist/swagger-ui.js.map && run-p --aggregate-output build-core build-bundle build-standalone build:es:bundle build:es:bundle:core"
```

Reply:
I had to tweak this script slightly. I was experiencing the `rimraf` command AFTER the build bundles.